metrics/isolator.hpp

```diff
@@ -170,6 +170,7 @@ class MetricsIsolator : public mesos::slave::Isolator
 } // namespace dcos {
 } // namespace mesosphere {
 
-extern mesos::modules::Module<mesos::slave::Isolator> com_mesosphere_dcos_MetricsIsolatorModule;
+extern "C" mesos::modules::Module<mesos::slave::Isolator>
```

Comment:
Do you have a good explanation why we need "C" linkage here? Is it something related to MSVC, to Windows... or to cmake? 

Can you add a comment or, in case you don't have a good explanation, a TODO so that we figure this out later?

Reply:
The reason of `extern "C"` is to have the specific name visible and as such have an ability to load module.

Reply:
right